R/tidyeval.R

```diff
@@ -145,6 +145,19 @@ dt_squash_call <- function(x, env, data, j = TRUE) {
     x
   } else if (is_call(x, "n", n = 0)) {
     quote(.N)
+  } else if (is_call(x, "n_distinct")) {
+    x <- match.call(dplyr::n_distinct, x, expand.dots = FALSE)
```

Comment:
Maybe use `call_match()` (from rlang) just for consistency?

Reply:
Looks like `call_match()` is new to the dev version of rlang. Should I add the dev version of rlang as a dependency?

And if yes, should I switch all of the other `match.call()` uses over to `call_match()` in this PR?

Reply:
Oh if it's only in dev version, don't worry about it.